README.md

```diff
@@ -77,7 +77,22 @@ You can now configure Googlmapper in a few simple steps. Open `app/config/packag
 
 ## Usage
 
-It's really as simple as using the Mapper class in any Controller / Model / File you see fit with:
+Initialize the map in your controller `MyController.php` as:
```

Comment:
Can you keep the sentence above, then add the code example under a ###Example title section.